lib/jekyll/converters/markdown.rb

```diff
@@ -56,21 +65,35 @@ def output_ext(ext)
       end
 
       def convert(content)
-        setup
-        @parser.convert(content)
+      setup
+        @parser.convert(
+          content
+        )
       end
 
       private
+      def get_custom_processor
+        md = @config["markdown"]
```

Comment:
`converter_name`, maybe? `md` confused me -- I thought it was supposed to be the markdown extension, e.g. `".md"` or `".markdown"`.